scripts/correlateTBW.py

```diff
@@ -18,7 +18,7 @@
 import argparse
 
 from lsl import astro
-from lsl.reader.ldp import LWA1DataFile
+from lsl.reader.ldp import LWA1DataFile, LWASVDataFile
```

Comment:
TBW is not available at LWA-SV.  The closest mode to TBW at LWA-SV is TBF (transient buffer - frequency domain).

Reply:
ok I'll get rid of the TBW changes